src/zapv2/spider.py

```diff
@@ -30,6 +30,7 @@ def __init__(self, zap):
     def status(self, scanid=None):
         """
         
+        This component is optional and therefore the API will only work if it is installed
```

Comment:
It's not a blocker, but any idea why this is dbl spacing?

Reply:
The previous line would contain the description of the endpoint, which probably does not have.

Reply:
Oh okay.